.gitignore

```diff
@@ -35,4 +35,7 @@ lerna-debug.log*
 !.vscode/extensions.json
 
 # misc
-/.silent-pay-indexer
\ No newline at end of file
+/.silent-pay-indexer
+
+#e2e 
+/.logs
```

Comment:
not required because we already have `*.log` on L7

Reply:
If you want to keep it still, it should be `/e2e/.logs/` not `/.logs/`

Reply:
yep, it was meant to be that :smiling_face_with_tear: